org-transclusion.el

```diff
@@ -254,6 +254,12 @@ specific keybindings; namely:
 
 (defvar org-transclusion-yank-remember-user-excluded-props '())
 
+(defvar org-transclusion-latex-preview-on-toggle nil
+  "Call org-latex-preview after transclusion-toggle.")
+
+(defvar org-transclusion-boundary nil
```

Comment:
For this (adding an extra empty line at the end of transclusion, I'd rather revert this change I made:

https://github.com/nobiot/org-transclusion/blob/3fb2a35e79c8542e2b9b7d03bbd75a9ed278e61a/org-transclusion.el#L1160-L1167

Reply:
That makes sense, this looks good to me.

Reply:
Should this be active in the feature branch? I do not see any changes regarding this point.

Reply:
It’s already in the main and merged with the feature branch too